File: ts/module-config/pull-with-push-override/fs/baz.ts
/*
 * Copyright © 2022 Paulo Villela. All rights reserved.
 * Use of this source code is governed by the MIT license
 * that can be found in the LICENSE file.
 */

import { makeCfgSrc } from "../config/cfg-src";

type BazCfgInfo = {
  w: string;
};

export const bazCfgSrc = makeCfgSrc<BazCfgInfo>(null);

export function baz() {
  console.log(bazCfgSrc().w.length);
}

File: go/module-config/pull-with-push-override/fs/baz.go
/*
 *  Copyright © 2021 Paulo Villela. All rights reserved.
 *  Use of this source code is governed by the Apache 2.0 license
 *  that can be found in the LICENSE file.
 */

package fs

import (
	"fmt"
	"github.com/pvillela/module-config/go/module-config/pull-with-push-override/config"
)

type BazCfgInfo struct {
	X string
}

type BazCfgSrc = config.CfgSrc[BazCfgInfo]

var BazCfgSrcV = config.MakeCfgSrc[BazCfgInfo](nil)

func Baz() {
	fmt.Println(len(BazCfgSrcV().X))
}
